include/phasar/PhasarLLVM/DataFlowSolver/Mono/Solver/InterMonoSolver.h

```diff
@@ -370,13 +423,19 @@ template <typename AnalysisDomainTy, unsigned K> class InterMonoSolver {
         if (!isIntraEdge(Edge)) {
           // real call
           for (auto &[Ctx, Facts] : Analysis[Src]) {
-            processCall(Edge); // TODO: decompose into processCall and
-                               // processCallToRet
+            if(isSensibleToSummarize(Src)){
+              mono_container_t summary =  summarize(ICF->getFunctionOf(Src),Facts);
```

Comment:
Once we got the summary, we should still store it and also must plug it in to model the effects of that function call.